sharding/__init__.py

```diff
@@ -0,0 +1,19 @@
+import pkg_resources
+
+from sharding.contracts.utils.smc_utils import (  # noqa: F401
+    get_smc_source_code,
+    get_smc_json,
```

Comment:
Any particular reason for them to be imported here?

Reply:
No particular, just have a glimpse of the tests and those functions seems are the only required APIs for the SMCHandler?